lib/pipefy_message/broker/aws/sns/publisher.rb

```diff
@@ -42,8 +42,8 @@ def do_publish(message, topic_name)
           result = topic.publish({ message: message.to_json, message_structure: " json " })
           @log.info(" Message Published with ID #{result.message_id}")
           result
-        rescue StandardError
-          @log.error("Failed to publish message [#{message}]")
+        rescue StandardError => e
+          @log.error("Failed to publish message [#{message}], error details: [#{e.inspect}]")
```

Comment:
💅 if you just need the message, you can do `e.message`

Reply:
Nice, by now I'll keep the `e.inspect` because I don't know if the message has everything that we need 😅